### ISSUE TITLE
Avoid intermediate file I/O

### ISSUE THREAD

**Issue by sajith** [atlanticwave-sdx/pce#58]
Library functions doing file I/O is not a good idea: disk I/O as an intermediate step will be likely inefficient; error handling can be complicated (how do we proceed when a file can't be opened?); it is easier to test functions that do not do I/O; it is likely to be surprise the library's users at runtime.  

PCE does file I/O in several places:

https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/Heuristic/Heur.py#L160
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/Heuristic/Heur.py#L164

https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/LB_Utilization_Solver.py#L15
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/LB_Utilization_Solver.py#L20
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/LB_Utilization_Solver.py#L57
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/LB_Utilization_Solver.py#L81
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/LB_Utilization_Solver.py#L108
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/LB_Utilization_Solver.py#L172

https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/MC_Solver.py#L34
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/MC_Solver.py#L57
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/MC_Solver.py#L86
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/MC_Solver.py#L154

https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/RandomTopologyGenerator.py#L65
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/RandomTopologyGenerator.py#L116
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/RandomTopologyGenerator.py#L128
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/RandomTopologyGenerator.py#L153
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/RandomTopologyGenerator.py#L172
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/RandomTopologyGenerator.py#L264
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/RandomTopologyGenerator.py#L271
https://github.com/atlanticwave-sdx/pce/blob/9c3be7f4355104eea7d97bba595522758fe9bbc3/src/LoadBalancing/RandomTopologyGenerator.py#L277

**Comment by YufengXin** [comment]
All of these functions will be phased out with the new implementation:
1. [pce/src/LoadBalancing/RandomTopologyGenerator.py] -> [pce/src/Utility/randomTopologyGenerator.py] and [pce/src/Utility/randomConnectionGenerator.py]
2. [pce/src/LoadBalancing/MC_Solver.py] and [pce/src/LoadBalancing/LB_Utilization_Solver.py]. -> [pce/src/LoadBalancing/TE_Solver.py]
3. [pce/src/Heuristic/Heur.py]: this one particular function will be removed.

**Comment by sajith** [comment]
No longer relevant after #83.
